app-admin/xstow/xstow-1.0.2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
+
+DESCRIPTION="replacement for GNU stow with extensions"
+HOMEPAGE="http://xstow.sourceforge.net/"
+SRC_URI="mirror://sourceforge/xstow/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ncurses"
+
+DEPEND="ncurses? ( sys-libs/ncurses:0= )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-ncurses.patch
```

Comment:
We usually define PATCHES=() in global scope instead (above src_prepare)

PATCHES=(
    "${FILESDIR}"/${P}-ncurses.patch
)

Reply:
done